### PR TITLE
hw01_Xuefei_Gao

### PR DESCRIPTION
**Pull request by IanGGG**
Here is a description of how I finish the homework.
1. I forked the hw01 repository and upload my photo.
2. I created a version control project in RStudio using the URL of the forked hw01 repository and pulled the repository.
3. I made changes to the README.md in RStudio and added some comments.
4. I pushed my project and checked if there are any mistakes in the preview of Github.
5. I edited the README.md file in RStudio based on what I found wrong in Github and re-pushed it.

Basically, the main difficulties come from learning the grammar of Markdown. At first, I ignored some spaces, which makes my list does not like a list. After I figure out some basic rules of Markdown, it is not that hard to use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01: Self-Introduction
+My name is **Xuefei Gao** and you can just call me **Ian**, which is easier to pronounce and remember. I am a raising second year and I plan to major in Math and Econ. I am also a fellow of SISRM and I am going to work with Professor Susan Stokes this summer.
+
+![Photo](Self_Portrait.png)
```
Comment (gmeyers405) [inline README.md:4]: Great job with adding a picture!

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+Here are some of my interests:
+1. Music
```
Comment (gmeyers405) [inline README.md:7]: I looked at the actual file and it seems like it had trouble processing the list within a list (i.e. the types of music you like). According to the Markdown guide (https://guides.github.com/features/mastering-markdown/), you need 4 spaces to indent your code, although it may also be possible that Markdown didn't like you using a bullet list within a numbered list. Despite this, I think it was a nice touch!

File: README.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
+  * Rock
+  * Classical
+2. Basketball
+3. Sneakers
+
+Here is my homepage of Github: https://github.com/IanGGG
```
Comment (gmeyers405) [inline README.md:13]: It doesn't really matter, since the link worked anyway, but if you wanted to hyperlink a word you could use the code [Github] (https://github.com/IanGGG).

File: README.md (hunk 2 above)
Comment (gmeyers405) [inline README.md:7]: I looked at the actual file and it seems like it had trouble processing the list within a list (i.e. the types of music you like). According to the Markdown guide (https://guides.github.com/features/mastering-markdown/), you need 4 spaces to indent your code, although it may also be possible that Markdown didn't like you using a bullet list within a numbered list. Despite this, I think it was a nice touch!

File: README.md (hunk 3 above)
Comment (gmeyers405) [inline README.md:13]: It doesn't really matter, since the link worked anyway, but if you wanted to hyperlink a word you could use the code [Github] (https://github.com/IanGGG).

**Review by gmeyers405 — COMMENTED** [review]
Overall, I think you did really well! You clearly know how to utilize the homework workflow, given how you've outlined it in your submission comment. You've also demonstrated Markdown proficiency using 4 elements: bold, image embedding, lists, and link embedding. You also made it clear who you were, what you were majoring in, your interests, and why you were taking the class.

I included a couple of comments about things I noticed in your README doc. To restate, it seems like you might have had some trouble indenting your music tastes within your interests list, which might be resolved by adding 4 spaces instead of hitting tab. You could also improve your code by using the Github Markdown Guide's syntax for embedding links, which allows you to hyperlink your website to a specific word. I guess a third thing would be to have headers (again, you hit the quota of 4, so it doesn't matter for the grade, but I think they would be a nice addition).

File: README.md [hunk 4]
```diff
@@ -1 +1,13 @@
...
+My name is **Xuefei Gao** and you can just call me **Ian**, which is easier to pronounce and remember. I am a raising second year and I plan to major in Math and Econ. I am also a fellow of SISRM and I am going to work with Professor Susan Stokes this summer.
+
+![Photo](Self_Portrait.png)
+
+Here are some of my interests:
+1. Music
+  * Rock
+  * Classical
+2. Basketball
+3. Sneakers
```
Comment (rammkripa) [inline README.md:11]: Sneakers really are the best! Are you into the retro or more modern ones?

**Review by rammkripa — COMMENTED** [review]
Cool Intro, Ian!
I think your use of markdown features like lists, bold, links, and images was nice.
What kinds of Rock music do you listen to?
Also, I have recently been made aware that it might be a good idea to have a separate images folder, especially if you're adding many pictures to your markdown file.
However, this seems like a relatively minor issue to me.
Good luck for the rest of the course and for SISRM!

File: README.md (hunk 2 above)
Comment (IanGGG) [reply README.md:7]: This is a really helpful advice. Thank you so much!

File: README.md (hunk 4 above)
Comment (IanGGG) [reply README.md:11]: I would say both. But recently I'm really into Converse. It's like going back to the good old days. :)

File: README.md (hunk 1 above)
Comment (nearridge) [inline README.md:4]: Nice job committing this to your repo.

File: README.md [hunk 5]
```diff
@@ -1 +1,13 @@
...
\ No newline at end of file
+# hw01: Self-Introduction
+My name is **Xuefei Gao** and you can just call me **Ian**, which is easier to pronounce and remember. I am a raising second year and I plan to major in Math and Econ. I am also a fellow of SISRM and I am going to work with Professor Susan Stokes this summer.
+
+![Photo](Self_Portrait.png)
+
+Here are some of my interests:
+1. Music
+  * Rock
+  * Classical
```
Comment (nearridge) [inline README.md:9]: Needs proper indenting.

File: README.md (hunk 3 above)
Comment (nearridge) [inline README.md:13]: Yeah that's the proper way to hyperlink.

**Review by nearridge — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |           |    x          |            |
| **Reproducibility**         |    x       |              |            |

Remarks: Great job uploading your images to GitHub. Remember, to properly hyperlink, you cannot just copy/paste – that doesn't signify to the markdown renderer that this is actually a link. You need to use the []() syntax.
